src/Serilog/Formatting/Display/MessageTemplateTextFormatter.cs

```diff
@@ -118,7 +118,13 @@ public void Format(LogEvent logEvent, TextWriter output)
                 }
 
                 if (pt.Alignment.HasValue)
+                {
+#if NETCOREAPP3_0_OR_GREATER
```

Comment:
Rather than `NETCOREAPP3_0_OR_GREATER` can we define an explanatory constant like `FEATURE_WRITE_STRINGBUILDER`? Existing ones are: https://github.com/serilog/serilog/blob/dev/src/Serilog/Serilog.csproj#L44

Reply:
done